android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormBaseActivity.java

```diff
@@ -124,6 +128,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             onActivityResultListeners.get(requestCode).onActivityResult(requestCode, resultCode, data);
         } else {
             super.onActivityResult(requestCode, resultCode, data);
+            //Get simprint result
+//            if(resultCode == Activity.RESULT_OK && requestCode == JsonFormConstants.ACTIVITY_REQUEST_CODE.REQUEST_CODE_REGISTER){
```

Comment:
Please remove this dead code if you don't intend to use it later.

Reply:
ok will remove. it was just like a snapet how to parse the simprints callback data

Reply:
okay.  I get it now